registration-processor/registration-processor-registration-status-service-impl/src/test/java/io/mosip/registration/processor/status/service/SyncRegistrationServiceTest.java

```diff
@@ -143,6 +143,9 @@ public void setup()
 		syncRegistrationDto.setPacketHashValue("ab123");
 		syncRegistrationDto.setSupervisorStatus("APPROVED");
 		syncRegistrationDto.setSyncType(SyncTypeDto.NEW.getValue());
+		syncRegistrationDto.setName("mosip");
+		syncRegistrationDto.setPhone("1234567890");
+		syncRegistrationDto.setEmail("mosip@gmail.com");

```

Comment:
New test cases to be added to SyncRegistrationDaoTest class and mainly SyncRegistrationServiceImpl since it is added with new log of generating the HMAC, hash code etc..

Reply:
taken care

Reply:
I see this is not taken care for SyncRegistrationServiceImpl class

Reply:
SyncRegistrationServiceTest in this class we have doing testing for SyncRegistrationServiceImpl